battleship.py

```diff
@@ -64,10 +64,10 @@ def mousePressed(data, event, board):
 def emptyGrid(rows, cols):
     grid=[]
     for i in range(rows):
-        l1 = []
+        cols = []
         for j in range(cols):
-            l1.append(EMPTY_UNCLICKED)
-        grid.append(l1)
+            cols.append(EMPTY_UNCLICKED)
+        grid.append(cols)

```

Comment:
Did you test the function after updating @Pravallika2812?
Please check and make the required changes in case there is an error.